spec/functional/builder_spec.rb

```diff
@@ -9,8 +9,10 @@ module Omnibus
     # for testing methods like +ruby+ and +rake+ without the need to compile
     # a real Ruby just for functional tests.
     #
+    # Haha, this will totally work on windows..  (no it won't).
```

Comment:
These types of comments are generally not helpful. Funny, but not helpful. Imagine a future engineer reading this. **Why** doesn't it work on Windows? Why the sarcasm? Can I refactor this?